### ISSUE TITLE
Latin American Spanish translation

### ISSUE THREAD

**Issue by kepp11** [ncase/mental-health#9]

**Comment by Desalambrade** [comment]
Hi! Will you use "lenguaje inclusivo" ("-e" non-binary/neutral gender mark) or at least the feminine instead of the masculine for universal sentences about people? It would be nice if the translation was inclusive.

**Comment by kepp11** [comment]
Yes! I will probably use "x" since the creator of the game would like the translations to use gender neutral pronouns.

**Comment by fantasmashy** [comment]
¿Como va la traducción? ¿Necesitan ayuda?

**Comment by kepp11** [comment]
¡Gracias!, ya está terminada; envié un correo a Nicky, pero aún no ha habido respuesta.
